mlibrary/pride
# Removing `Pride.FieldTree.Special` and `Pride.FieldTree.Tag`.

# Overview
`Pride.FieldTree.Special` and `Pride.FieldTree.Tag` and never called in Pride nor Search, so they are getting removed. The `README` for `Pride.FieldTree` is also getting removed as it uses `Tag` as an example, which will now no longer exist.

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
   * Break the new/updated unit tests to make sure they're working properly.
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#pride-fieldtree-special-tag-removal` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#pride-fieldtree-special-tag-removal"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- Make sure the PR is consistent in these browsers:
